js/web/lib/onnxjs/backends/webgl/inference-handler.ts

```diff
@@ -32,7 +32,7 @@ export class WebGLInferenceHandler implements InferenceHandler {
     let artifact = this.session.programManager.getArtifact(op);
     if (!artifact) {
       const programInfo = op.createProgramInfo(this, inputs);
-      artifact = this.session.programManager.build(programInfo);
+      artifact = this.session.programManager.build(programInfo, op.constructor.name);
```

Comment:
should remove second parameter. since we already have `ProgramInfo.name` as optional property, we should use it.

Reply:
done